Assets/MixedRealityToolkit/_Core/Devices/OpenVR/OpenVRDeviceManager.cs

```diff
@@ -41,15 +42,14 @@ public override void Enable()
 
         public override void Update()
         {
-            GenericOpenVRController controller;
+            GenericOpenVRController controller = null;
             InputTracking.GetNodeStates(nodeStates);
             for (int i = 0; i < nodeStates.Count; i++)
             {
                 if (IsNodeTypeSupported(nodeStates[i]) && activeControllers.ContainsKey(nodeStates[i].nodeType))
                 {
-                    //TODO - Need logic to determine controller type (if possible)
-                    controller = activeControllers[nodeStates[i].nodeType] as GenericOpenVRController;
-                    if(controller.Enabled) controller.UpdateController(nodeStates[i]);
+                    controller = activeControllers[nodeStates[i].nodeType];
+                    if (controller != null && controller.Enabled) controller.UpdateController(nodeStates[i]);
```

Comment:
missing braces.

Reply:
Will check.

Reply:
`controller?.Enabled` :)

Reply:
fixed

Reply:
Sadly @Jarodshow that wouldn't work here, as controller is not nullable. (I had tried that already :P)

Reply:
`controller?.Enabled == true` doesn't work?